setup.sh

```diff
@@ -33,7 +33,21 @@ if [[ ! -x /usr/local/bin/ansible ]]; then
   brew install ansible
 fi
 
+if [[ ! -x /usr/local/bin/bash ]]; then
```

Comment:
why nod add these to https://github.com/ePages-de/mac-dev-setup/blob/master/common/tasks/main.yml#L8-L27?